node/packages/parser/src/cosmos/types.ts

```diff
@@ -0,0 +1,38 @@
+export type Network = 'mainnet'
+
+interface Value {
+  amount: string
+  denom: string
```

Comment:
`denomination`?

Reply:
Original key name from cosmos-sdk that I am passing back through unchained, going to leave it as.